tda/client.py

```diff
@@ -1085,6 +1030,128 @@ def update_watchlist(self, account_id, watchlist_id, watchlist_spec):
         are valid.
         `Official documentation
         <https://developer.tdameritrade.com/watchlist/apis/patch/accounts/
-        %7BaccountId%7D/watchlists/%7BwatchlistId%7D-0>`__.'''
+        %7BaccountId%7D/watchlists/%7BwatchlistId%7D-0>`_.'''
         path = '/v1/accounts/{}/watchlists/{}'.format(account_id, watchlist_id)
-        return self.__patch_request(path, watchlist_spec)
+        return self._patch_request(path, watchlist_spec)
+
+class SyncClient(BaseClient):
```

Comment:
Let's maybe rename this as `Client` both for backwards compatibility/similarity and for simplicity?